src/custom/components/AccountDetails/index.tsx

```diff
@@ -176,7 +176,7 @@ export default function AccountDetails({
               <AccountGroupingRow>
                 {formatConnectorName(connector, walletInfo)}
                 <div>
-                  {connector !== injected && connector !== walletlink && (
+                  {/* connector !== injected && connector !== walletlink && (
```

Comment:
In this case (not a Mod), better remove it completely.
Unless you are commenting it out for some other reason?

Reply:
we may want it later